tests/0_common/15_list_repos.sh

```diff
@@ -2,6 +2,6 @@
 
 t_Log "Running $0 - Showing the repos we have configured"
 
-yum -d0 repolist -v
+yum -d0 repollist -v
```

Comment:
This is a failure introduced for testing purposes and won't be present non WIP pr